notebooks/203-meter-reader/203-meter-reader-digital.ipynb

```diff
@@ -0,0 +1,1059 @@
+    "2. Information to be mapped (keyword: 'MP'): 'k/m' may be recognized as km, so km can be mapped as' k/m'.\n",
+    "3. Addition decimal point (keyword 'AD'): Sometimes decimal points may be missed. If the number of decimal places is known in advance, decimal points can be added."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cbc138ac-73f5-46d4-abb6-a67dcf6e9ccd",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Post-processing, fix some error made in recognition\n",
+    "def post_processing(results, post_configration):\n",
+    "    \"\"\"\n",
+    "    Postprocessing function for correcting the recognition errors.\n",
+    "    Parameters:\n",
+    "        results (Dict): The result directory.\n",
+    "        post_configration (Dict): The configuration directory.\n",
+    "    \"\"\"\n",
+    "    for key in post_configration.keys():\n",
+    "        if len(post_configration[key]) == 0:\n",
+    "            continue  # nothing to do\n",
+    "        for post_item in post_configration[key]:\n",
+    "            key_word = post_item[0]\n",
+    "            if key_word == 'MP':  # mapping\n",
+    "                source_word = post_item[1]\n",
+    "                target_word = post_item[2]\n",
+    "                if source_word in results[key]:\n",
+    "                    results[key] = target_word\n",
+    "            elif key_word == 'RP':  # removing\n",
+    "                source_word = post_item[1]\n",
+    "                target_word = post_item[2]\n",
+    "                results[key] = results[key].replace(source_word, target_word)\n",
+    "            elif key_word == 'AD':  # add point\n",
+    "                add_position = post_item[1]\n",
+    "                results[key] = results[key][:add_position] + '.' + results[key][add_position:]\n",
+    "    return results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4545be5e-a358-442a-8640-08d139f752c5",
+   "metadata": {},
+   "source": [
+    "After estabilishing the function, only a configuration dictionary needs to be passed in for post-processing, such as `['MP', 'LF', '探头:LF-01']`, `['RP', 'X', ':']`, `['AD', -2]`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f8c55f0-d688-4932-9c95-89cab4b7be56",
+   "metadata": {},
+   "source": [
+    "### Main Function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ecb1666f-1b7d-4d70-9622-ee9d63595923",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# the input of recognition should be image, DESIGN information, compiled_model\n",
+    "def main_function(img, DESIGN_LAYOUT, RESULT_TEMP, rec_compiled_model, rec_output_layer, text_decoder, preprocess_function=None):\n",
```

Comment:
Since rec_compiled_model, rec_output_layer, text_decoder are reused, I dont think we should set them as a parameter in function definition. Just use it ?